app/uploaders/configurable_uploader_versions.rb

```diff
@@ -0,0 +1,30 @@
+module ConfigurableUploaderVersions
```

Comment:
:+1: wasn't sure if it was a conscious decision for the uploaders or not.  Just namespaced it and will push another commit to namepace the other uploaders.